src/4.3_to_5.0/migrate_43_to_50.ts

```diff
@@ -434,19 +434,6 @@ export async function migrate_43_to_50(
             });
             counters.widgets.success++;
           } catch (error) {
-            if (error instanceof PartialMigrationError) {
```

Comment:
Does it make a difference to move this block after the call to `_addErrorToReport`? I don't mind, but just trying to understand the change.